scripts/install.sh

```diff
@@ -51,38 +51,63 @@ else
     echo -e "${ORANGE}Please look up how to install Python 3.9 or later for your specific operating system.${NC}"
     exit 1
 fi
-
 #
-# Install needed system packages
+# Get os package manager
 #
-# Run apt-get update
-echo -e "${GREEN}Updating apt-get cache...${NC}"
-sudo apt-get update || exit 1
+if [[ -f /etc/os-release ]]; then
+    source /etc/os-release
+    if [[ "$ID_LIKE" == *"debian"* ]]; then
+        pkg_manager="apt-get"
```

Comment:
Could be nice to add an echo here saying we detected Debian-based OS, same for arch

Reply:
Done ;)